frontend/src/components/LandingPage/Tools/Tools.test.js

```diff
@@ -0,0 +1,23 @@
+import { render, screen } from '@testing-library/react';
```

Comment:
I see that you used a different testing library. I havent used this before but following your tests I can see your logic. Great work Nick.